src/assets/styles/base.scss

```diff
@@ -164,21 +164,12 @@ p {
   font-weight: 600;
 }
 
-.fade-out {
+// class to auto-hide loading containers after 2 seconds
+.auto-fade-out {
   animation: fadeOut 2s forwards;
 }
 
-// styles for 'fade leave' transition
-// NB: 'fade enter' looks better without transition
-// .fade-enter-active,
-.fade-leave-active {
```

Comment:
These obsolete styles were use for the transition spinners in the filing views (eg, AR, COD). I changed those to `v-fade-transition` which handles this automatically.